nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListFile.java

```diff
@@ -177,7 +175,7 @@ private void runNext() throws InterruptedException {
 
 
     @Test
-    @Ignore("Intended only for manual testing, as is very expensive to run as a unit test. Performs listing of 1,000,000 files (doesn't actually create the files, though - injects them in) to " +
+    @Disabled("Intended only for manual testing, as is very expensive to run as a unit test. Performs listing of 1,000,000 files (doesn't actually create the files, though - injects them in) to " +
         "ensure performance is not harmed")
```

Comment:
This should be changed to `EnableIfSystemProperty` so that it could be run optionally, instead of disabled.

Reply:
going to set it to run on nifi.test.performance then

Reply:
on further testing it appears that the performance of this test (even on main) is inconsistent... let me know what you'd recommend doing with this!